Resources/Prototypes/_DV/Entities/Mobs/Player/ovinia.yml

```
@@ -0,0 +1,31 @@
+- type: entity
+  name: Urist McSheepy
+  parent: MobOviniaBase
+  id: MobOvinia
+  components:
+  - type: CombatMode
+  - type: MindContainer
+    showExamineInfo: true
+  - type: Input
+    context: "human"
+  - type: Speech
+    speechVerb: Ovinia
+  - type: MobMover
```

SpeechComponent is already on MobOviniaBase and can be removed from here
```suggestion
    context: "human"
  - type: MobMover
```